rust/private/rust.bzl

```diff
@@ -48,6 +48,10 @@ See https://github.com/bazelbuild/rules_rust/pull/203 for more details
 def _determine_output_hash(lib_rs):
     return repr(hash(lib_rs.path))
 
+def _deprecated_attributes(ctx):
+    if getattr(ctx.attr, "out_dir_tar", None):
+        fail(ctx, "`out_dir_tar` is deprecated, please use cargo/cargo_build_script.bzl instead.")
```

Comment:
I don't know if it makes sense to mention using cargo raze > 0.3.4. I think most of the uses of this comes from templates predating that version.

Reply:
Done.